DesignSafe-CI/portal
# Task/DES-2115 - Versioning and Amends Metadata Managers

## Overview: ##

This PR adds refactored metadata management modals for the Projects area, metadata schema changes, and updates to support versioning and amendment capabilities for Field Research projects. 

## PR Status: ##


## Related Jira tickets: ##

* [DES-2115](https://jira.tacc.utexas.edu/browse/DES-2115)
* [DES-2299](https://jira.tacc.utexas.edu/browse/DES-2299) 

## Summary of Changes: ##

* There will be group testing sessions for this PR
* Many of the Add/Edit [Metadata] modals have been refactored and adjusted to work with versioning and amendments
* Related Works and Associated Projects fields have been added to each publishable type of metadata (projects, missions, experiments etc...); the types that receive dois.
* Conditionally required fields are handled differently and more consistently across the different metadata management modals.

## Testing Steps: ##

1. Set up or edit an existing project of any type
2. Ensure that editing saves/updates as expected
3. Check for any visual bugs/inconsistencies in the projects area
